build.sbt

```diff
@@ -1,6 +1,9 @@
 import ReleaseTransformations._
 import microsites.ExtraMdFileConfig
 
+addCompilerPlugin("org.scalameta"  % "semanticdb-scalac" % "4.0.0" cross CrossVersion.full)
```

Comment:
Do we need this enabled going forward?

Reply:
it's needed for scalafix, I like to keep scalafix available, but I'm fine to clean up if you want to

Reply:
This did the trick for 90% of the code: 
```scalafix https://gist.githubusercontent.com/erikwj/1c576d0f78a095dc7ff2e14ebb71c8a9/raw/c139a6b92f9ee3a7b9edfa7dd6856dcff0010afa/SeqToList```

Reply:
Right. I just wasn't sure what's the reason to keep it around, until we have another rewrite. I'm voting for omitting it.

Reply:
Getting back to this. @erikwj, can we omit this for now (as it's no longer needed)?

Reply:
ok @vkostyukov will submit the changes